### ISSUE TITLE
dropdown: toggle and item target size

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2670]
## Description of the issue

Dropdown toggle button and item target sizes should be a 24px minimum, or  [WCAG 2.2 2.5.8 AA compliant](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#258-target-size-minimum-aa).

### Impacted component(s)

- `pf-dropdown`
- `pf-dropdown-item`

### Steps to reproduce

1. Go to [Dropdown documentation](https://patternflyelements.org/components/dropdown/)
2. Inspect the toggle button.
3. Inspect the items.

### Expected behavior

Minimum width and height should be 24px.

**Comment by bennypowers** [comment]
we can't diverge from upstream styles. a11y issues on pf should be filed there

**Comment by bennypowers** [comment]
@nikkimk can we close this one?
